16-slices/exercises/03-slice-literal/main.go

```diff
@@ -50,15 +50,20 @@ func main() {
 		alives    []bool    // Up/Down status of web servers
 	)
 
-	names = []string{}
-	distances = []int{}
-	data = []byte{}
-	ratios = []float64{}
-	alives = []bool{}
+	names = []string{"A", "B", "C"}
+	distances = []int{3, 10, 8, 4, 0}
+	data = []byte{8, 2, 7, 0, 5}
+	ratios = []float64{1.01, 141.76}
+	alives = []bool{true, true, true, false}
+	var s string
```

Comment:
declare the variable just before where it is used

Reply:
fixed